react/src/components/BookList/components/BookItem/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import default_book from '../../../../assets/default_book.svg';
+
+class BookItem extends Component {
+  render() {
+    let image_url
+    if(this.props.image_url === null){
+      image_url = default_book;
```

Comment:
We could use `defaultProps` and use `this.props.image_url` directly:

```
class BookItem extends Component {
  ...
}

BookItem.defaultProps = {
  image_url: default_book
};

export default BookItem;
```

Reply:
image_url's value is null when an image URL is not defined, and defaultProps is only used when the value is undefined. Is there any workaround i'm not seeing?